nobl9/release_channel_schema.go

```diff
@@ -0,0 +1,28 @@
+package nobl9
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const releaseChannel = "release_channel"
+
+func schemaReleaseChannel() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Release channel of the created datasource [stable/beta]",
```

Comment:
maybe add `Default` option, pointing at `stable`? It'd be aligned with what we do at backend --> setting stable channel when none was provided

Reply:
On the backend, we can customize the default release channel for different organizations, so I would not put default values here.

Reply:
oh, didn't know that, makes sense :)